packages/salesforcedx-vscode-core/src/commands/forceApexTestRunCodeAction.ts

```diff
@@ -79,7 +79,8 @@ export class ForceApexTestRunCodeActionExecutor extends SfdxCommandletExecutor<{
       .withFlag('--tests', this.test)
       .withFlag('--resultformat', 'human')
       .withArg('--synchronous')
-      .withFlag('--loglevel', 'error');
+      .withFlag('--loglevel', 'error')
+      .withLogName('force_apex_test_run_codeaction');
```

Comment:
Should this be 'force_apex_test_run_code_action'?

Reply:
Yeah, let me change that